data/templates/sangoma.tmpl

```diff
@@ -183,7 +183,7 @@
         <P237 para="FirmwareUpGrade.ConfigServerPath">{{ provisioning_url2 }}</P237>
         <P1145 para="FirmwareUpGrade.AllowDHCPOption">66</P1145>
         <P145 para="FirmwareUpGrade.ToOverrideServer">0</P145>
-        <P194 para="FirmwareUpGrade.AutoUpgrade">{{ provisioning_complete ? '0' : '1' }}</P194>
+        <P194 para="FirmwareUpGrade.AutoUpgrade">{{ ( provisioning_complete and provisioning_freq == 'never' ) ? '0' : '1' }}</P194>
```

Comment:
If we don't have any argument to keep provisioning_complete, let's remove it completely


```suggestion
        <P194 para="FirmwareUpGrade.AutoUpgrade">{{ provisioning_freq == 'never'  ? '0' : '1' }}</P194>
```

Reply:
Please ignore my comment above